.github/workflows/package-fedora.yml

```diff
@@ -0,0 +1,38 @@
+name: Package GEDKeeper3 for Fedora
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  DOTNET_VERSION: '6.0.*'
+
+jobs:
+  build-fedora:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Init
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - name: .NET SDK check
+      run: dotnet sdk check
+# error NETSDK1084: There is no application host available for the specified RuntimeIdentifier 'freebsd-x64'
+# https://learn.microsoft.com/en-us/dotnet/devops/dotnet-build-github-action
+#    - name: Install dependencies
+#      run: dotnet restore
+#      working-directory: projects/GKv3
+    - name: Build
+      run: dotnet build -c Linux_Release
+      working-directory: projects/GKv3
+    - name: Package
+      run: bash ./gk_linux_rpm_fedora_package.sh
```

Comment:
For build through `Actions`, you can substitute some other, abstract version, such as `3X`. But from the assembly scripts that I use locally for my own purposes, there is no need to remove version.